qiskit/providers/aer/backends/pulse_simulator.py

```diff
@@ -48,7 +48,8 @@ class PulseSimulator(AerBackend):
         'max_shots': 50000,
         'description': 'A pulse-based Hamiltonian simulator',
         'gates': [],
-        'basis_gates': []
+        'basis_gates': [],
+        'use_cpp_ode_func': True  # Use the C++ ODE func or the Cyhton one
```

Comment:
typo `Cython`

I feel like this should be in `backend_options`, not the config, if you expect a user to change it.

Reply:
I wasn't thinking about the user, but the fact that I want both methdos (Cython/C++ Solvers) coexist for a while. Only the C++ solver will eventually remain. But you got a point, I'd rather make it easier for the user to change it, so I'm moving it to the backend_options.